.travis.yml

```diff
@@ -96,3 +96,10 @@ matrix:
       script:
       - "scripts/build_assembly_no_test.sh scalding-core"
       - "scripts/test_typed_tutorials.sh"
+
+    - scala: 2.10.4
+      env: BUILD="test execution tutorials"
+      script:
+      - "scripts/build_assembly_no_test.sh scalding-core"
```

Comment:
do we still need this with the fixes you put in place in the bash script? I'm trying to minimize the time the tests take here.